subsystem/drivetrain.py

```diff
@@ -95,15 +90,13 @@ def get_drive_motor_traveled_distance(self) -> meters:
         if self.drive_reversed:
             sensor_position *= -1
 
-        return (
-            sensor_position - self.start_dist
-        ) / constants.drivetrain_move_gear_ratio_as_rotations_per_meter
+        return sensor_position / constants.drivetrain_move_gear_ratio_as_rotations_per_meter
 
     def get_turn_motor_angle(self) -> radians:
```

Comment:
Why does this do the same thing as get_current_motor_angle? If we really need them to do the same thing, then one should just return the result of the other, so that the code only has to be maintained in one place. Or do we need to refactor something?